internal/httpproxy/http_proxy.go

```diff
@@ -45,13 +45,12 @@ const (
 )
 
 type Config struct {
-	CA                string
 	TLSCert           string
 	TLSKey            string
 	K8sAPIServerToken string
-	K8sAPIServerURL   string
```

Comment:
We don't use this field anymore (we use the resource's address in GAT instead) but I forgot to remove this.